packages/tizen_rpc_port/lib/tizen_rpc_port.dart

```diff
@@ -0,0 +1,7 @@
+library tizen_rpc_port;
+
+export 'parcel.dart';
+export 'parcelable.dart';
+export 'port.dart';
+export 'proxy_base.dart';
+export 'stub_base.dart';
```

Comment:
To make `tizen_rpc_port.dart` the only entry point of this plugin, you may move these files into a directory named `src`.

Reply:
When I try your suggestion, I failed it.
```dart
library tizen_rpc_port;

export '../src/parcel.dart';   // fail
export 'src/parcelable.dart';   // fail
export 'src/port.dart';
export 'src/proxy_base.dart';
export 'src/stub_base.dart';
```

It seems to forcecd remove "../" symbols,
`'../lib/src/parcel.dart': No such file or directory`
How should I this?

Reply:
I've tested it. It works fine.
Please check it again.

Reply:
Ah, I misunderstood.
When I make `src` folder in `lib`, it works fine.
Thanks :)